lms/djangoapps/class_dashboard/dashboard_data.py

```diff
@@ -22,12 +25,14 @@
 ]
 
 
-def get_problem_grade_distribution(course_id):
+def get_problem_grade_distribution(course_id, enrollment):
     """
     Returns the grade distribution per problem for the course
 
     `course_id` the course ID for the course interested in
 
```

Comment:
Nit: I think preferred style is to place no empty line between arguments in docstring.